docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.9"
```

Comment:
I am trying to understand the version here. I get "ERROR: Version in "./docker-compose.yml" is unsupported." while being on Docker version 20.10.7 and docker-compose version 1.25.0. 1.25.0 seems to be old based on information here https://docs.docker.com/compose/compose-file/ so I need to upgrade. I am on a 20.04 LTS system. Would it be possible to support older systems for the docker configuration files?